### DISCUSSION TITLE
Allowing HEAD only formulae on taps?

### DISCUSSION THREAD

**Discussion by technicalpickles** [Homebrew/discussions#3749] [Tap maintenance and Homebrew development]
I'm working on a private tap for some formulae we need or have made modifications for. One of the formulaes doesn't have proper releases, so it has to be head-only. Unfortunately, `brew test-bot --only-tap-syntax` fails with:

```
gusto/gusto/wait-for-it:
  * Head-only (no stable download)
Error: 1 problem in 1 formula detected
```

Is there a way add an exception for this? I see there is a `audit_exceptions` directory, but it's not clear to me what exceptions are possible to be included. I've found this, but it doesn't look like there's exceptions for it: https://github.com/Homebrew/brew/blob/4441452f4e7a7efdd5ae8cc98466d8cf6fa410ab/Library/Homebrew/formula_auditor.rb#L590

I could comment out the GitHub Action steps for `brew test-bot --only-tap-syntax`, but I'd definitely loose out on other useful audits.

**Comment by ipatch** [comment]
did you up patching homebrew? or did you find a fix / work around for this issue?
